data/json/items/biosignatures.json

```diff
@@ -71,5 +71,26 @@
       "charges_to_use": 1,
       "effects": [ { "id": "fetid_goop", "duration": 3600 } ]
     }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "insect_dust",
+    "name": { "str_sp": "chitin dust" },
+    "category": "chems",
+    "weight": "40 g",
+    "color": "white",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -3,
+    "healthy": -10,
+    "calories": 26,
```

Comment:
You can't eat other sources of exochitin barring wastebread, you shouldn't be able to get calories out of flakes.

Reply:
changed to matched chitin chunk attributes